README.md

```diff
@@ -14,7 +14,14 @@ A `Node.js` based command-line client for [tldr](https://github.com/tldr-pages/t
 ## Installing
 
 ```bash
-$ npm install -g tldr
+npm install -g tldr
+```
+
+If yo have troubles with post-install script, try the following commands:
```

Comment:
- "yo" --> "you"
- "troubles" --> "trouble"
- "with post-install script" --> "with the post-install script"